app/controllers/home_controller.rb

```diff
@@ -1,7 +1,13 @@
 class HomeController < ApplicationController
   before_action :set_user
 
-  def index; end
+  def index
+    @search = params[:search]
+    return @all_books = Book.all.order(:author) if @search.blank?
```

Comment:
What do you think about renaming `all_books` to just `books`? `all_books` implies it's going to hold every book in the DB, and we know that will not be the case when a search filter is applied.

Reply:
That's a good point, thanks! I changed it.